src/app/clusters/ota-requestor/DefaultOTARequestorDriver.h

```diff
@@ -109,7 +109,10 @@ class DefaultOTARequestorDriver : public OTARequestorDriver
     uint16_t maxDownloadBlockSize  = 1024;
     // Maximum number of times to retry a BUSY OTA provider before moving to the next available one
     static constexpr uint8_t kMaxBusyProviderRetryCount = 3;
-    uint8_t mProviderRetryCount; // Track retry count for the current provider
+    // Track retry count for the current provider
+    uint8_t mProviderRetryCount;
```

Comment:
consider providing an initial value like the members above. Likely a ` = 0` suffices.

Reply:
Done.